src/components/greeting.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+
+const Greeting = () => {
+  const message = useSelector((state) => state.greetings.greating);
```

Comment:
Nice work so far. However, instead of typing `greetings.greeting`, you entered `greetings.greating` on line 5 above.  This is a typo (typing error) causing the message variable to be undefined. Kindly correct the typo to fix this error.  :pray: